frontend/test/metabase/scenarios/question/reproductions/13097-mongo-apply-distinct-count-multiple-columns.js

```diff
@@ -0,0 +1,38 @@
+import { restore, visualize } from "__support__/e2e/cypress";
+
+const MONGO_DB_NAME = "QA Mongo4";
+
+export function issue13097() {
+  describe.skip("issue 13097", () => {
```

Comment:
Should we still be skipping?

Reply:
If the issue hasn't been fixed - we should. This doesn't change that functionality.

Now, regarding this specific issue, it seems that Cam fixed it long time ago. I'll have to check was there a specific reason why the associated repro hasn't been unskipped, or did we simply forget to do it.

Reply:
https://github.com/metabase/metabase/pull/21477